nova/virt/vmwareapi/vmops.py

```diff
@@ -154,6 +165,7 @@ def __init__(self, session, virtapi, volumeops, cluster=None,
         self._datastore_browser_mapping = {}
         self._imagecache = imagecache.ImageCacheManager(self._session,
                                                         self._base_folder)
+        self._session_cls = session_cls
```

Comment:
This is to avoid inter-dependencies.
The `driver` package holds `VMwareAPISession` class.
Since the `driver` is the main entrypoint which then depends on `vmops`, I thought to avoid pulling any imports from `driver` back into `vmops`.